site/en/community/contribute/code_style.md

```diff
@@ -11,24 +11,21 @@ and use [pylint](https://www.pylint.org/) to check your Python changes.
 
 ### pylint
 
-To install `pylint` and retrieve TensorFlow's custom style definition:
+To install `pylint`:
 
 ```bash
-
 $ pip install pylint
-$ wget -O /tmp/pylintrc https://raw.githubusercontent.com/tensorflow/tensorflow/master/tensorflow/tools/ci_build/pylintrc
-
 ```
 
-To check a file with `pylint`:
+To check a file with `pylint` from the TensorFlow source code root directory:
 
 ```bash
-$ pylint --rcfile=/tmp/pylintrc myfile.py
+$ pylint --rcfile=tensorflow/tools/ci_build/pylintrc tensorflow/python/keras/losses.py
 ```
 
 ### Supported Python versions
 
-TensorFlow supports Python >= 3.5. See the
+TensorFlow supports Python 3.6-3.8. See the
 [installation guide](https://www.tensorflow.org/install) for details.
```

Comment:
```suggestion
For supported Python versions, see the TensorFlow
[installation guide](https://www.tensorflow.org/install).
```

Thanks. Let's remove the version from this page so we don't have to update it down the road.